applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c

```diff
@@ -72,6 +72,19 @@ static void bond_find(const struct bt_bond_info *info, void *user_data)
 	}
 }
 
+static void filter_accept_list_add(const struct bt_bond_info *info, void *user_data)
+{
+	int ret;
+
+	if (IS_ENABLED(CONFIG_BT_FILTER_ACCEPT_LIST)) {
```

Comment:
This is only called when CONFIG_BT_FILTER_ACCEPT_LIST is enabled. I'd argue we either remove this check or return an error if it is called when this flag is not set.

Reply:
It is to guard bt_le_filter_accept_list_add from the linker if FAL is not enabled, but I'll test to see if it's needed after I moved the code below

Reply:
Removed